standard-covoiturage_openapi.yaml

```diff
@@ -176,6 +176,43 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
+  /journeys:
+    get:
+      tags:
+      - List
+      summary: Get journeys by their ids.
+      description: Route used to retrieve journeys using their identifiers.
+      operationId: getJourneys
+      parameters:
+        - in: query
+          name: ids[]
+          required: true
+          explode: true
+          schema:
+            type: array
+            items:
+              type: string
+              minLength: 1
+              maxLength: 255
+      responses:
+        200:
+          description: Ok. Request processed successfully.
+          content:
+            application/json:
+              schema:
```

Comment:
This endpoint takes several ids and returns a single journey - I believe you meant to return an array here, right?